### PR TITLE
Update command-line-tools.md

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/administration/command-line-tools.md [hunk 1]
```diff
@@ -71,6 +75,30 @@ COMMANDS:
         Example:
             platform -assign_role -email="user@example.com" -role="system_admin"
 
+    -join_channel                     Joins a user to the channel.  It requires the -email, channel_name and
```
Comment (lfbrock) [inline source/administration/command-line-tools.md]: Should it be -channel_name?

File: source/administration/command-line-tools.md [hunk 2]
```diff
@@ -71,6 +75,30 @@ COMMANDS:
...
+                                      -team_name flags.  You may need to logout of your current session
+                                      for the new channel to be applied.  Requires an enterprise license.
+        Example:
+            platform -join_channel -email="user@example.com" -team_name="name" -channel_name="channel_name"
+
+    -leave_channel                     Removes a user from the channel.  It requires the -email, channel_name and
```
Comment (lfbrock) [inline source/administration/command-line-tools.md]: Same here, -channel_name?

**Comment by lfbrock** [comment]
+1 other than small comments

**Comment by jasonblais** [comment]
Typo fixed, great catch

**Comment by esethna** [comment]
+1

**Pull request merged**